great_expectations/data_context/store/gx_cloud_store_backend.py

```diff
@@ -267,6 +267,16 @@ def _get(self, key: Tuple[str, ...]) -> ResponsePayload:  # type: ignore[overrid
     def _move(self) -> None:  # type: ignore[override]
         pass
 
+    def update(self, key, value, **kwargs):
+        """
+        Essentially `set` but validates that a given key-value pair does already exist.
+        """
+        if not self.has_key(key):
+            raise StoreBackendError(
+                f"Store does not have a value associated the following key: {key}."
+            )
+        return self._update(id=key[1], value=value)
```

Comment:
Is this overriding an inherited method?

Reply:
So this actually seems a bit different than I originally thought. 
I thought the difference was just selecting the `key[1]` item but the overridden method actually calls `self.set()` not `self._update()`
```python
    def update(self, key, value, **kwargs):
        """
        Essentially `set` but validates that a given key-value pair does already exist.
        """
        if not self.has_key(key):
            raise StoreBackendError(
                f"Store does not have a value associated the following key: {key}."
            )
        return self.set(key=key, value=value, **kwargs)
```